virtual_ecosystem/example_data/generation_scripts/soil_example_data.py

```diff
@@ -35,6 +35,10 @@
 # Generate a range of plausible values (0.1-1.0) for the POM pool [kg C m^-3].
 pom_values = 0.1 + 0.9 * (gradient) / (64)
 
+# Generate a range of plausible values (0.00015-0.0005) for the microbial necromass pool
+# [kg C m^-3].
+necromass_values = 0.00015 + 0.00035 * (gradient) / (64)
```

Comment:
Not sure what the brackets are for:

```suggestion
necromass_values = 0.00015 + 0.00035 * gradient / 64.0
```

It's probs also best practice to make the 64 explicitly a floating-point number.